tools/github/Add-AzsdkMilestones.ps1

```diff
@@ -36,13 +40,17 @@ if ($PSCmdlet.ParameterSetName -eq 'Languages') {
     }
 }
 
+if ($PSCmdlet.ParameterSetName -eq 'RepositoryFile') {
+    $Repositories = Get-Content $RepositoryFilePath
+}
+
 $date = $StartDate
 $milestones = do {
     # Start with the first of the month at 23:59 UTC.
     $date = [DateTimeOffset]::Parse("$($date.ToString('yyyy-MM'))-01T23:59:59Z")
 
-    # The end date is always the first Friday of next month.
-    $next = $date.AddMonths(1)
+    # The end date is always the first Friday of the month.
+    $next = $date
```

Comment:
Just use `$date` then. Doesn't matter that we bump the days because we only use the year and month in the title. Less to keep track of.